docs/topics/js/js-to-kotlin-interop.md

```diff
@@ -156,35 +156,81 @@ C.Companion.callNonStatic(); // The only way it works
 It's also possible to apply the `@JsStatic` annotation to a property of an object or a companion object, making its getter
 and setter methods static members in that object or the class containing the companion object.
 
+### Usage of `BigInt` type to represent Kotlin's `Long` type
+<primary-label ref="experimental-general"/>
+
+Kotlin/JS uses JavaScript's built-in `BigInt` type to represent Kotlin's `Long` values when compiling to modern JavaScript (ES2020).
+
+To enable the usage of `BigInt` type, you need to add the following compiler option to your
+`build.gradle(.kts)` file:
+
+```kotlin
+// build.gradle.kts
+kotlin {
+    js {
+        ...
+        compilerOptions {
+            freeCompilerArgs.add("-Xes-long-as-bigint")
+        }
+    }
+}
+```
+
+This feature is [Experimental](components-stability.md#stability-levels-explained). Please
+report any problems in our issue tracker, [YouTrack](https://youtrack.jetbrains.com/issue/KT-57128/KJS-Use-BigInt-to-represent-Long-values-in-ES6-mode).
```

Comment:
```suggestion
This feature is [Experimental](components-stability.md#stability-levels-explained). Share you feedback in our [issue tracker](https://youtrack.jetbrains.com/issue/KT-57128/KJS-Use-BigInt-to-represent-Long-values-in-ES6-mode).
```
Removing "please" as we don't need to be so polite in our documentation :) I also suggested a more neutral phrase when asking for feedback.